tests/test_validation.py

```diff
@@ -73,6 +73,65 @@ def setup_1bin_shapesys(
     }
 
 
+@pytest.fixture(scope='module')
+def spec_1bin_lumi(source=source_1bin_example1()):
+    spec = {
+        'channels': [
+            {
+                'name': 'singlechannel',
+                'samples': [
+                    {
+                        'name': 'signal',
+                        'data': source['bindata']['sig'],
+                        'modifiers': [
+                            {'name': 'mu', 'type': 'normfactor', 'data': None}
+                        ],
+                    },
+                    {
+                        'name': 'background',
+                        'data': source['bindata']['bkg'],
+                        'modifiers': [{'name': 'lumi', 'type': 'lumi', 'data': None}],
```

Comment:
don't we need to associate some data with the modifier? like the scale of the uncertainty?

Reply:
Yep, I guess this goes into the auxdata -- but I'm still not sure where it goes yet. `readxml` doesn't have an easy way to access it and copy it across all modifiers.